easybuild/easyconfigs/a/Amber/Amber-16-2016.11-CrayGNU-2016.11-cuda-8.0.eb

```diff
@@ -0,0 +1,52 @@
+# @author: karakasv gppezzi
+# @author: victor holanda rusu
+
+easyblock = 'MakeCp'
+
+name = 'Amber'
+version = '16-2016.11'
+cudaversion = '8.0'
+versionsuffix = '-cuda-%s' % cudaversion
+amberversion = '16'
+
+homepage = 'http://ambermd.org/'
+description = 'Assisted Model Building with Energy Refinement'
+
+toolchain = {'name': 'CrayGNU', 'version': '2016.11'}
+toolchainopts = { 'verbose' : False }
+#sources = ['Amber%(version)s.tar.bz2', 'AmberTools%(version)s.tar.bz2']
+sources = ['Amber%(version)s.tar.bz2']
```

Comment:
Could you please add as comment here the commands you did for patching this version?